src/phasorpy/phasor.py

```diff
@@ -387,34 +390,37 @@ def phasor_semicircle(
     imag *= 0.5
     return real, imag
 
-
-def phasor_calibrate(
-    real: ArrayLike,
-    imag: ArrayLike,
-    phase0: ArrayLike = 0.0,
-    modulation0: ArrayLike = 1.0,
-    /,
-) -> tuple[NDArray[Any], NDArray[Any]]:
-    """Return calibrated/referenced phasor coordinates.
+def phasor_calibrate(real, imag, reference_real, reference_imag, frequency, lifetime, fraction, **kwargs):
+    """
+    Return calibrated/referenced phasor coordinates.
 
     Calibration of phasor coordinates in fluorescence lifetime analysis is
     necessary to account for the instrument response function (IRF) and delays
     in the electronics.
 
-    This function can also be used to transform/rotate any phasor coordinate.
-
     Parameters
     ----------
     real : array_like
-        Real component of phasor coordinates to calibrate.
+        Real component of phasor coordinates to be calibrated.
     imag : array_like
-        Imaginary component of phasor coordinates to calibrate.
-    phase0 : array_like, optional
-        Angular component of polar coordinates for calibration in radians.
-        Defaults to 0.0.
-    modulation0 : array_like, optional
-        Radial component of polar coordinates for calibration.
-        Defaults to 1.0.
+        Imaginary component of phasor coordinates to be calibrated.
+    reference_real : array_like
+        Real component of phasor coordinates from reference of known lifetime.
+    reference_imag : array_like
+        Imaginary component of phasor coordinates from reference of known 
+        lifetime.
+    frequency : array_like
+        Laser pulse or modulation frequency in MHz.
+        A scalar or one-dimensional sequence.
+    lifetime : array_like
+        Lifetime components in ns. See notes below for allowed dimensions.
+    fraction : array_like, optional
+        Fractional intensities or pre-exponential amplitudes of the lifetime
+        components. Fractions are normalized to sum to 1.
+        See notes below for allowed dimensions.
+    **kwargs
+        Additional keyword arguments passed to :py:func:`phasor_center`
+        and :py:func:`phasor_from_lifetime`,
```

Comment:
I think it makes sense in this case to explicitly list the few parameters and repeat the definitions. This makes it easier for the user.

Reply:
I agree, as they are not that many arguments, it won't be that long the list of arguments that it ends up being a mess. Maybe if for some reason the other functions are changed and more arguments are added we may need to revise this.